src/compute-client/src/controller.rs

```diff
@@ -216,6 +216,15 @@ impl<T> ComputeController<T> {
             storage,
         }
     }
+
+    /// List compute objects that depend on the given object.
+    pub fn reverse_compute_dependencies(
+        &self,
+        id: GlobalId,
+        instance_id: ComputeInstanceId,
```

Comment:
Bike-shedding time:
* To make this consistent with the other controller methods, could you make `instance_id` the second argument?
* In the context of the controller, we talk about "collections" rather than "objects".
* It's a bit redundant to have "compute" in the name here, since we are in the context of the compute controller anyway.

How about calling this method `collection_reverse_dependencies` or `collection_dependants`?

Reply:
> To make this consistent with the other controller methods, could you make instance_id the second argument?

Done

> In the context of the controller, we talk about "collections" rather than "objects".

Changed.

> It's a bit redundant to have "compute" in the name here, since we are in the context of the compute controller anyway. How about calling this method collection_reverse_dependencies or collection_dependants?

I renamed to `collection_reverse_dependencies`. (And I also renamed the wrapper method in the `ComputeController`.